schemas/app-sre/escalation-policy-1.yml

```diff
@@ -45,6 +45,9 @@ properties:
       jiraComponent:
         type: string
         description: component to set on jira tickets
+      jiraCustomFieldsJSON:
```

Comment:
If this is only used in ACS tickets context, it looks weird that it is something that apparently can affect any ticket. If we feel that this may evolve in the future, then I think it makes more sense for it to have a name specific to ACS.